packages/nextjs/utils/scaffold-eth/networks.ts

```diff
@@ -1,15 +1,19 @@
+import { useEffect, useState } from "react";
 import * as chains from "viem/chains";
+import { useNetwork } from "wagmi";
 import scaffoldConfig from "~~/scaffold.config";
 
-export type TChainAttributes = {
+export type ChainAttributes = {
```

Comment:
For consistency, we probably want to remove all the other `T` prefixes in the codebase

Opened: https://github.com/scaffold-eth/scaffold-eth-2/issues/617